sidebar/src/components/interact/contracts/params/InteractParams.logic.ts

```diff
@@ -19,8 +19,8 @@ export const useInteractParams = (contracts: InteractContracts) => {
   const displayParams = inputs && inputs.length > 0;
 
   useEffect(() => {
-    form.resetField('inputs');
-  }, [selectedFunction, selectedContractAddress]);
+      form.resetField('inputs');
+  }, [selectedFunction, selectedContractAddress, form]);
```

Comment:
Avoid using form in dependencies, otherwise the inputs will be reset every time form change and we don't want that